module1/homework/make_unique/include/make_unique.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <memory>
+#include <type_traits>
+
+namespace cs {
+template <typename T,
+          typename... Args,
+          typename = std::enable_if_t<!std::is_array_v<T>>>
```

Comment:
great idea to pass it as template argument

Reply:
+1